HerPublicWebsite.BusinessLogic/Models/AnonymisedReport.cs

```diff
@@ -0,0 +1,35 @@
+﻿using HerPublicWebsite.BusinessLogic.Models.Enums;
+
+using HerPublicWebsite.BusinessLogic.Extensions;
+
+namespace HerPublicWebsite.BusinessLogic.Models;
+
+public class AnonymisedReport
+{
+    public string PostcodeFirstHalf { get; set; }
```

Comment:
Please add a column for the date and time of the submission - we may need it for generating the export to the DMS